src/components/Verse/VerseActionsMenu.tsx

```diff
@@ -20,6 +19,13 @@ import styles from './VerseActionsMenu.module.scss';
 import Modal from '../dls/ModalNew/Modal';
 import VerseAdvancedCopy from './AdvancedCopy/VerseAdvancedCopy';
 
+interface Verse {
+  textUthmani: string;
+  verseKey: string;
+  chapterId: number | string;
+  verseNumber: number;
+}
```

Comment:
idt this should live in verse Actions, until you name VerseActions :D

Reply:
`Verse` here is a sub-set of `Verse` that only contains the data required for the `VerseActionsMenu` to work.

Reply:
`Verse` here is a sub-set of `Verse` that only contains the data required for the `VerseActionsMenu` to work. Rename to `ActionsVerse`.

Reply:
`Verse` here is a sub-set of `Verse` that only contains the data required for the `VerseActionsMenu` to work. Renamed to `ActionsVerse`.